### PR TITLE
add workwechat msgtype

### PR DESCRIPTION
**Pull request by bitqiu**
## Checklist

- [x] I have reviewed the [contributing guidelines](https://github.com/jertel/elastalert2/blob/master/CONTRIBUTING.md).
- [x] I have included unit tests for my changes or additions.
- [x] I have successfully run `make test-docker` with my changes.
- [x] I have manually tested all relevant modes of the change in this PR.
- [x] I have updated the [documentation](https://elastalert2.readthedocs.io).
- [x] I have updated the [changelog](https://github.com/jertel/elastalert2/blob/master/CHANGELOG.md).

### CODE REVIEW

**Comment by nsano-rururu** [comment]
Please add this line to the New features section at the top of `CHANGELOG.md`:

```
- [workwechat] add workwechat msgtype - [#1369](https://github.com/jertel/elastalert2/pull/1369) - @bitqiu
```

**Comment by bitqiu** [comment]
Already added

> Please add this line to the New features section at the top of `CHANGELOG.md`:
> 
> ```
> - [workwechat] add workwechat msgtype - [#1369](https://github.com/jertel/elastalert2/pull/1369) - @bitqiu
> ```

already done

**Comment by nsano-rururu** [comment]
I apologize for the lack of explanation. Please add it below what is currently listed.

current

```
- [workwechat] add workwechat msgtype - [#1369](https://github.com/jertel/elastalert2/pull/1369) - @bitqiu
- Add workwechat alerter - [#1367](https://github.com/jertel/elastalert2/pull/1367) - @wufeiqun
```

fix

```
- Add workwechat alerter - [#1367](https://github.com/jertel/elastalert2/pull/1367) - @wufeiqun
- [workwechat] add workwechat msgtype - [#1369](https://github.com/jertel/elastalert2/pull/1369) - @bitqiu
```

**Comment by bitqiu** [comment]
> - [workwechat] add workwechat msgtype - [#1369](https://github.com/jertel/elastalert2/pull/1369) - @bitqiu

already done

**Comment by nsano-rururu** [comment]
Has test code been added when work_wechat_msgtype is set to markdown?

File: elastalert/schema.yaml [hunk 1]
```diff
@@ -795,6 +795,7 @@ properties:
 
   ### WorkWechat
   work_wechat_bot_id: { type: string }
+  work_wechat_msgtype: { type: string }
```
Comment (nsano-rururu) [inline elastalert/schema.yaml:798]:
```yaml
work_wechat_msgtype: { type: string }
```

to

```yaml
work_wechat_msgtype: {type: string, enum: ['text', 'markdown']}
```

**Comment by nsano-rururu** [comment]
@jertel 
I think this pull request can be merged. I don't think there are any particular problems with the functionality or test code.

**Review by jertel — APPROVED** [review]

**Review by jertel — APPROVED** [review]

**Pull request merged**